sunpy/coordinates/ephemeris.py

```diff
@@ -43,8 +47,25 @@ def get_body_heliographic_stonyhurst(body, time='now'):
     """
     obstime = parse_time(time)
 
-    body_icrs = ICRS(get_body_barycentric(body, obstime))
-    body_hgs = body_icrs.transform_to(HGS(obstime=obstime))
+    if observer is None:
+        body_icrs = get_body_barycentric(body, obstime)
+    else:
+        observer_icrs = SkyCoord(observer).icrs.cartesian
+
+        # This implementation is modeled after Astropy's `_get_apparent_body_position`
+        light_travel_time = 0.*u.s
```

Comment:
```suggestion
        light_travel_time = 0. * u.s
```

Reply:
Is it our style guidelines to have spaces around the "*" even for units?  I see it done both ways in our codebase.  I prefer not having the spaces for such constants, but I'll do whatever is our guideline.

Reply:
Yeah, I used to not but then my linter nagged me into it. I wonder if @nabobalis has legislated for this.

Reply:
I'm not sure it's an internal rule but I think it's a PEP8 issue? I'm sure flake8 tells me to put spaces.

Reply:
It might be worth making an explicit note for units, because as @ayshih points out sometimes it feels weird to put the spaces around the operator before units.

Reply:
> Thus the PEP, in its current form, does not dictate whether or not you should use spaces around the + operator (or other arithmetic operators like * and / and **). You are free to "use your own judgement".

So we should pick one?

Reply:
Ok, I think we should move this to a new issue.